.github/workflows/main.yml

```diff
@@ -18,6 +18,63 @@ env:
   OIKOTIE_VENDOR_ID: test
   ELASTICSEARCH_VERSION: '7.9.1'
   ELASTICSEARCH_HOST: localhost
+  PUBLIC_PGP_KEY: |
```

Comment:
This change can be found also in here: https://github.com/City-of-Helsinki/apartment-application-service/pull/81/commits. Which one is correct?

Reply:
This is because I cherry-picked this commit from #81. The commit ID was different because it's calculated based on the tree, and since that commit was last in the changeset, the ID was different.

I've rearranged the commits in the original PR and now the commit IDs are the same.